diff --git a/lib/LoremIpsumBundle/src/DependencyInjection/KnpULoremIpsumExtension.php b/lib/LoremIpsumBundle/src/DependencyInjection/KnpULoremIpsumExtension.php
new file mode 100644
index 0000000..adadd2c
--- /dev/null
+++ b/lib/LoremIpsumBundle/src/DependencyInjection/KnpULoremIpsumExtension.php
@@ -0,0 +1,14 @@
+<?php
+
+namespace KnpU\LoremIpsumBundle\DependencyInjection;
+
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\HttpKernel\DependencyInjection\Extension;
+
+class KnpULoremIpsumExtension extends Extension
+{
+    public function load(array $configs, ContainerBuilder $container)
+    {
+        var_dump('We\'re alive!');die;
+    }
+}
