diff --git a/LoremIpsumBundle/tests/FunctionalTest.php b/LoremIpsumBundle/tests/FunctionalTest.php
index 922e076..8f06970 100644
--- a/LoremIpsumBundle/tests/FunctionalTest.php
+++ b/LoremIpsumBundle/tests/FunctionalTest.php
@@ -62,6 +62,11 @@ class KnpULoremIpsumTestingKernel extends Kernel
             $container->loadFromExtension('knpu_lorem_ipsum', $this->knpUIpsumConfig);
         });
     }
+
+    public function getCacheDir()
+    {
+        return __DIR__.'/cache/'.spl_object_hash($this);
+    }
 }
 
 class StubWordList implements WordProviderInterface
