diff --git a/LoremIpsumBundle/tests/FunctionalTest.php b/LoremIpsumBundle/tests/FunctionalTest.php
new file mode 100644
index 0000000..a2c5792
--- /dev/null
+++ b/LoremIpsumBundle/tests/FunctionalTest.php
@@ -0,0 +1,26 @@
+<?php
+
+namespace KnpU\LoremIpsumBundle\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Config\Loader\LoaderInterface;
+use Symfony\Component\HttpKernel\Kernel;
+
+class FunctionalTest extends TestCase
+{
+    public function testServiceWiring()
+    {
+
+    }
+}
+
+class KnpULoremIpsumTestingKernel extends Kernel
+{
+    public function registerBundles()
+    {
+    }
+
+    public function registerContainerConfiguration(LoaderInterface $loader)
+    {
+    }
+}
