diff --git a/LoremIpsumBundle/src/DependencyInjection/Compiler/WordProviderCompilerPass.php b/LoremIpsumBundle/src/DependencyInjection/Compiler/WordProviderCompilerPass.php
index 7f2cf7d..5a79fff 100644
--- a/LoremIpsumBundle/src/DependencyInjection/Compiler/WordProviderCompilerPass.php
+++ b/LoremIpsumBundle/src/DependencyInjection/Compiler/WordProviderCompilerPass.php
@@ -4,14 +4,18 @@ namespace KnpU\LoremIpsumBundle\DependencyInjection\Compiler;
 
 use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Reference;
 
 class WordProviderCompilerPass implements CompilerPassInterface
 {
     public function process(ContainerBuilder $container)
     {
+        $definition = $container->getDefinition('knpu_lorem_ipsum.knpu_ipsum');
+        $references = [];
         foreach ($container->findTaggedServiceIds('knpu_ipsum_word_provider') as $id => $tags) {
-            var_dump($id);
+            $references[] = new Reference($id);
         }
-        die;
+
+        $definition->setArgument(0, $references);
     }
 }
\ No newline at end of file
